samples/samples-outofproc/OutputBindingSamples/QueueTriggerProducts.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.Common;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Extension.Sql;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.OutputBindingSamples
+{
+    public static class QueueTriggerProducts
+    {
+        [Function("QueueTriggerProducts")]
+        [SqlOutput("[dbo].[Products]", ConnectionStringSetting = "SqlConnectionString")]
+        public static List<Product> Run([QueueTrigger("testqueue")] string queueMessage)
```

Comment:
out of curiosity what is queueMessage used for? I see that it's an unused parameter, just curious in general for queue triggers

Reply:
We use the [queue trigger](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-queue-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-csharp) to start a function when a new item is received on a queue. The queue message is provided as input to the function. More like event based triggers on sql, here we are checking for a message on queue and then executing the function.